Library/Formula/docker-compose.rb

```diff
@@ -0,0 +1,79 @@
+class DockerCompose < Formula
+  homepage "http://www.fig.sh/"
+  url "https://github.com/docker/compose/archive/1.1.0.tar.gz"
+  sha1 "175066934c19f455606b16f1b4e4b9f26fc3f599"
+
+  conflicts_with "fig", :because => "Branding change from Fig to Docker Compose."
+
+  bottle do
+    cellar :any
+    revision 1
+    sha1 "af332540bdc40e6e574d92ca7169c71071af9a5d" => :yosemite
+    sha1 "1f29c6947f4d6a797041dae82d0e5e2463224043" => :mavericks
+    sha1 "53a11121643db837d8d684859e9141643366f63c" => :mountain_lion
+  end
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on "libyaml"
+  depends_on "docker" => :recommended
+  depends_on "boot2docker" => :recommended
```

Comment:
yes, it's possible to install Docker and boot2docker manually, that's why I just left it as recommended.